staff/ana-sanjuan/app-template/home.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Home</title>
+</head>
+<body>
+    <header>
+        <h1>Home</h1>
+        <nav>
+            <a href="home.html">Home</a> |
+            <a href="news.html">News</a> |
+            <a href="about.html">About</a> |
+            <a href="contact.html">Contact</a>|
+            <a href="register.html">Register</a> |
+            <a href="login.html">Login</a> |
+            <a href="profile.html">Profile</a> |
+        </nav>
+    </header>
+    <hr>
+
+    <section>
+        <article>
+            <header>
+                <img src="https://placekitten.com/200/200" alt="cute kitten">
+                <h2>Lorem title</h2>
+            </header>
+            <article>
```

Comment:
I think is confusing to put an article inside an article

Reply:
yes, I have been rethinking the code, and I think I need to use main and put sections inside instead of articles inside articles...  thanks for your comment!!

Reply:
Maybe article is duplicated?

Reply:
In the news page I do repeat the articles, but here I think is better to change to sections. 
Thanks for your comment Almudena! :)